himalaya.el

```diff
@@ -125,15 +125,18 @@
   :group 'himalaya)
 
 
-(defvar-local himalaya-mailbox nil
+(defvar himalaya-mailbox nil
   "The current mailbox.")
 
-(defvar-local himalaya-account nil
+(defvar himalaya-account nil
   "The current account.")
 
-(defvar-local himalaya-uid nil
+(defvar himalaya-uid nil
   "The current message uid.")
 
+(defvar himalaya-reply nil
+  "True if the current message is a reply.")
+
```

Comment:
I needed to replace `defvar-local` by `defvar` because it seemed that variables were not accessible from `himalaya-send-buffer`. I do not know the impact of such a change?